File: vy-android-app/app/src/main/java/com/lemon/vy3000/ui/home/SearchViewModel.java
package com.lemon.vy3000.ui.home;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class SearchViewModel extends ViewModel {

    private MutableLiveData<String> mText;

    public SearchViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("Search");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: vy-android-app/app/src/main/java/com/lemon/vy3000/ui/fragments/search/SearchViewModel.kt
package com.lemon.vy3000.ui.fragments.search

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class SearchViewModel : ViewModel() {
    private val mText: MutableLiveData<String> = MutableLiveData()
    val text: LiveData<String>
        get() = mText

    init {
        mText.value = "This is search fragment"
    }
}